### PR TITLE
[W5.7][W10-1] Way Tan

### PR DESCRIPTION
**Pull request by waytan**
Edit find function to be non-case sensitive. Update tests and User Guide accordingly.

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -4,12 +4,42 @@
...
+        assertNoNull(1, 2, 3, 0.1, "ab", true, '2');
+    }
+
+    private void assertHasNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```
Comment (damithc) [inline test/java/seedu/addressbook/common/UtilsTest.java:42]: A quick response: this code should not be part of this PR i.e., don't mix multiple enhancements in the same PR

**Comment by waytan** [comment]
Hi Prof, noted on the PR. I may have branched at the wrong place, so wasn't sure how to rectify it. (the commit for the earlier LO was also captured)

**Comment by damithc** [comment]
> Hi Prof, noted on the PR. I may have branched at the wrong place, so wasn't sure how to rectify it. (the commit for the earlier LO was also captured)

One way to fix it:
1. close this PR
1. create a new branch at the right place
1. cherry-pick the relevant commits from this branch to the new branch
1. open a new PR with the new branch

**Comment by kzzj217** [comment]
I feel that this improvement can be good and user-friendly, just try to fix the issue. :)
